workshops/test/base.py

```diff
@@ -231,7 +231,7 @@ def _setUpEvents(self):
         event_end = today
         Event.objects.create(start=event_start,
                              end=event_end,
-                             slug='ends_today',
+                             slug='ends_today-ongoing',
```

Comment:
as above - if it's just a join-up dash, please use an underscore (it's confusing to read)